Sources/SwiftInspectorVisitors/ProtocolVisitor.swift

```diff
@@ -56,16 +56,18 @@ public final class ProtocolVisitor: SyntaxVisitor {
     let typealiasVisitor = TypealiasVisitor(parentType: .simple(name: name))
     typealiasVisitor.walk(node.members)
 
+    let propertiesVisitor = PropertySyntaxVisitor()
+    propertiesVisitor.walk(node.members)
```

Comment:
one of these days it'd be great if we could consolidate our `TypealiasVisitor`, `PropertySyntaxVisitor`, and `AssociatedtypeVisitor` visitors into a single `MemberVisitor` that could be fed a `MemberDeclBlockSyntax` and return all the types we need.

Not today. But... should I file an issue for that? Seems like it'd be a perf win.

Reply:
Oh that's interesting. I think that'd be good yeah 😉

Reply:
https://github.com/fdiaz/SwiftInspector/issues/101